src/admin.go

```diff
@@ -230,7 +388,252 @@ func UpdateRoute(options Options, dist cloudfront.DistributionSummary) error {
 	return nil
 }
 
+/*
+* Find the best matching certificates to the domain name.
+* return the best matching certificate if a certificate exists
+* (see https://github.com/EagerIO/Stout/issues/20#issuecomment-232174716)
+* or, if there is no certificate with the domain requested, return ""
+ */
+func findMatchingCertificate(options Options, acmService *acm.ACM) (string, error) {
+
+	// list issued and pending certificates
+	certificatesResponse, err := acmService.ListCertificates(&acm.ListCertificatesInput{
+		CertificateStatuses: []*string{
+			amazonaws.String("ISSUED"),
+			amazonaws.String("PENDING_VALIDATION"),
+		},
+	})
+	if err != nil {
+		return "", err
+	}
+	//if there are no certificates, return nil
```

Comment:
Would you mind making the comments a bit more consistent? (space after the `//`, start with a capital letter, period if appropriate).